### PR TITLE
[APPS-1339] Use the panoply fork of dbstream

### PR DESCRIPTION
**Pull request by yonivy**
https://panoply.atlassian.net/browse/APPS-1339

Before this change can be deployed the changes in https://panoply.atlassian.net/browse/APPS-1351 must be deployed, otherwise we risk breaking production (which now simply points to latest master).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "dbstream-mongo",
-  "version": "1.4.1",
+  "version": "1.5.0",
```
Comment (german-panoply) [inline package.json:3]: The current release version is `v2.0.0`, don't we wanna have something like `2.1.0` here?

Screen Shot 2022-08-31 at 6 42 26 PM

```suggestion
  "version": "2.1.0",
```

File: package.json [hunk 2]
```diff
@@ -21,15 +21,15 @@
...
   },
-  "homepage": "https://github.com/avinoamr/dbstream-mongo",
+  "homepage": "https://github.com/panoplyio/dbstream-mongo",
   "devDependencies": {
     "mocha": "^1.21.0",
     "sift": "0.0.18"
   },
   "dependencies": {
-    "dbstream": "^1.0.1",
+    "dbstream": "https://github.com/panoplyio/dbstream#v1.0.3",
```
Comment (german-panoply) [inline package.json]: Not sure what format is preferred, this or the one with `github:org/repo`, but have you noticed that in some places we add the `git+` prefix? Not sure what it does, but please take a look:

Screen Shot 2022-08-31 at 6 45 21 PM

File: package.json (hunk 1 above)
Comment (german-panoply) [reply package.json:3]: Also, it looks like the the `package-lock.json` file had been generated before you changed the version here:

Screen Shot 2022-08-31 at 6 46 01 PM

File: package.json (hunk 2 above)
Comment (yonivy) [reply package.json]: The [npm install docs](https://docs.npmjs.com/cli/v8/commands/npm-install) state that the prefix `git:` is used to identify node modules hosted in a git repo. 

Using only `https://` is working properly, I was able to install all and it was reflected well in the lock file. But, it seems that npm is "recommending" we specify it's a git repo. I'll update the deps to use `git+https:` and see if it's working properly locally. BTW the `github:` prefix is specific to github but seems to behave as the `git:` prefix.

**Review by german-panoply — APPROVED** [review]

**Pull request closed without merging**